### PR TITLE
[Merged by Bors] - [CU-33mm7ge] add blockchain-integrations as Subsquid's code owner

### PR DESCRIPTION
**Pull request by andresmechali**
https://app.clickup.com/t/33mm7ge

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-33mm7ge Add blockchain-integrations as codeowner of Subsquid](https://app.clickup.com/t/33mm7ge)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -38,6 +38,7 @@ README.md @ComposableFi/technical-writing-reviewers @ComposableFi/core @Composab
 ## Frontend and Blockchain Integration
 
 frontend/ @ComposableFi/blockchain-integrations @ComposableFi/parachain-finance
+subsquid/ @ComposableFi/blockchain-integrations
```
Comment (vimukthi-git) [inline .github/CODEOWNERS:41]: There is another line in this file for subsquid, you can modify it and add the new group

Comment (andresmechali) [reply .github/CODEOWNERS]: You are right, I didn't see it

**Review by vimukthi-git — APPROVED** [review]

**Review by vivekvpandya — APPROVED** [review]

**Pull request closed without merging**